src/handlers/poster-hubble/components/OverlayHubble.tsx

```diff
@@ -0,0 +1,27 @@
+import { OverlayProps } from '../../poster-base';
+import HubbleProgressBar from './HubbleProgressBar';
+import HubbleOrderView from './HubbleOrderView';
+
+export default function OverlayHubble({
+  // poster,
+  // posterTitle,
```

Comment:
?

Reply:
Poster will come when poster.color is thing, posterTitle can indeed be removed

Reply:
Removed posterTitle